dspace/config/registries/schema-publicationIssue-types.xml

```diff
@@ -27,5 +27,149 @@
             delimited by commas.
         </scope_note>
     </dc-type>
+    
+    <!-- OpenAIRE4 Guidelines -->
+    <!-- 6.2.2. Attribute relatedIdentifierType (M) -->
+    <!-- specific type required by OpenAIRE4 Guidelines -->
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>ark</qualifier>
+        <scope_note>Archival Resource Key</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>arxiv</qualifier>
+        <scope_note>arXiv.org identifier</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>bibcode</qualifier>
+        <scope_note>Astrophysics Data System bibliographic codes; bibcodes can be resolved via 
+            http://adsabs.harvard.edu/abs/bibcode.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>doi</qualifier>
+        <scope_note>Digital Object Identifier</scope_note>
```

Comment:
Question for today's Entities meeting:  Should we create a new field for DOIs?  Currently DSpace is storing DOIs in `dc.identifier.uri` (alongside Handles).  Or, is this field meant for *externally created* DOIs (e.g. if a publisher has created a DOI as well)?

Reply:
We actually store DOIs in two fields:  dc.identifier.uri and dc.identifier.  The choice seems to hinge on whether DSpace manages the identifier (the former) or the identifier was entered manually (the latter).  There are two Jira issues about this:  DS-3708 linked to DS-2199.